int3face/assetlist.json

```diff
@@ -0,0 +1,173 @@
+{
+  "$schema": "../assetlist.schema.json",
+  "chain_name": "int3face",
+  "assets": [
+    {
+      "description": "Int3face blockchain native token",
+      "denom_units": [
+        {
+          "denom": "uint3",
+          "exponent": 0
+        },
+        {
+          "denom": "int3",
+          "exponent": 6
+        }
+      ],
+      "base": "uint3",
+      "name": "Int3face Token",
+      "display": "int3",
+      "symbol": "INT3",
+      "logo_URIs": {
+        "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/int3face/images/int3.png",
+        "svg": "https://raw.githubusercontent.com/cosmos/chain-registry/master/int3face/images/int3.svg"
+      },
+      "images": [
+        {
+          "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/int3face/images/int3.png",
```

Comment:
Is this logo meant to be round? Or is it intentional to have the black background making it square?

Reply:
Changed the logo, it's now in a circular shape